features/step_definitions/api_docs_step.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-Given(/^provider "(.*?)" has a swagger 1.0$/) do | org_name |
+Given "provider {string} has a swagger 1.0" do |org_name|
```

Comment:
This is much easier to read without looking up transformation definitions. It's good to have this step and `I should see the provider key of {provider}` use same approach be it the one or the other.

Reply:
Same with the rest of steps using `provider {string}` approach.